supervisor/const.py

```diff
@@ -63,6 +63,9 @@
 CONTENT_TYPE_URL = "application/x-www-form-urlencoded"
 COOKIE_INGRESS = "ingress_session"
 
+HEADER_FORWARDED_HOST = "X-Forwarded-Host"
+HEADER_HOST = "Host"
+HEADER_REFERER = "Referer"
```

Comment:
Let's use for the official headers the hdrs module as we do on core and other parts i.e. (https://github.com/home-assistant/supervisor/blob/dev/supervisor/api/ingress.py#L8)